travis/run_pylint.py

```diff
@@ -67,7 +41,9 @@ def run_pylint(paths, cfg, beta_msgs=None, sys_paths=None, extra_params=None):
     sys.path.extend(sys_paths)
     cmd = ['--rcfile=' + cfg]
     cmd.extend(extra_params)
-    subpaths = get_subpaths(paths)
+
+    subpaths = getaddons.get_subpaths(paths)
```

Comment:
I am building knowledge on MQT!!!
I prepared a script to run the travis tests locally, hopefully this will help me to debug.
I will publish the scripts once I successfully pass travis tests in the PR as well.